internal/venice-common/src/test/java/com/linkedin/venice/store/rocksdb/TestRocksDBUtils.java

```diff
@@ -0,0 +1,112 @@
+package com.linkedin.venice.store.rocksdb;
+
+import static com.linkedin.venice.store.rocksdb.RocksDBUtils.deletePartitionDir;
+import static org.testng.Assert.assertFalse;
+import static org.testng.Assert.assertTrue;
+
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.Comparator;
+import org.testng.annotations.Test;
+
+
+public class TestRocksDBUtils {
+  @Test
+  public void testDeletePartitionDir() throws IOException {
+    Path baseDir = null;
+
+    try {
+      // tmp directory "rocksdb/storeName/1/1/{file1.txt|file2.txt}"
+      baseDir = Files.createTempDirectory("rocksdb");
```

Comment:
You don't need these duplicated codes if you can use   @BeforeMethod and   @AfterMethod to do setup and clean up accordingly.